tests/kernel/smp/Kconfig

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2020 Intel Corporation.
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
This kconfig controls the configuration of kernel code, you can't put it in a test!  This belongs in kernel/Kconfig, right after SCHED_IPI_SUPPORTED.

Reply:
Thanks for correcting this. Add this CONFIG into kernel/Kconfig.